test/test.config.js

```diff
@@ -3,6 +3,7 @@
  */
 import {config} from '@bedrock/core';
 import {fileURLToPath} from 'node:url';
+import https from 'node:https';
```

Comment:
probably should change this to use `@bedrock/https-agent`

Reply:
changed to use `@bedrock/https-agent` here: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/44/commits/b36fa044b364dbbc4e33bac206fc655083514e36